metricbeat/module/kafka/partition/partition.go

```diff
@@ -48,7 +48,7 @@ var errFailQueryOffset = errors.New("operation failed")
 // New creates a new instance of the partition MetricSet.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	opts := kafka.MetricSetOptions{
-		Version: "3.6.0",
```

Comment:
One comment here as well. Decision behind 2.1.0 and why we should ideally change this without research.